Graphs/Table/index.js

```diff
@@ -282,6 +282,11 @@ class Table extends AbstractGraph {
             }
         })
 
+        if (filteredColumns.length) {
+            filteredColumns[0].firstColStyle = firstColToolTipStyle;
+            _.last(filteredColumns).lastColStyle = lastColToolTipStyle;
```

Comment:
is '_' imported? Please just import what is required only from lodash

Reply:
@natabal done.